apps/ejabberd/src/mod_blocking.erl

```diff
@@ -47,16 +47,17 @@ process_iq_get(_, _From = #jid{luser = LUser, lserver = LServer},
               E ->
                   {error, E}
           end,
-    case Res of
-        {ok, Lst} ->
-            {result, blocking_query_response(Lst)};
-        {error, _} ->
-            {error, ?ERR_INTERNAL_SERVER_ERROR}
-    end;
+    IqRes = case Res of
+                {ok, Lst} ->
+                    {result, blocking_query_response(Lst)};
+                {error, _} ->
+                    {error, ?ERR_INTERNAL_SERVER_ERROR}
```

Comment:
I'm more and more convinced that hook implementations should return `{ok, Acc}` in success cases and sth like `{error, Reason, Acc}`. This subject of a separate PR though.